doc/classes/Resource.xml

```diff
@@ -71,7 +71,7 @@
 				Returns the [RID] of this resource (or an empty RID). Many resources (such as [Texture2D], [Mesh], and so on) are high-level abstractions of resources stored in a specialized server ([DisplayServer], [RenderingServer], etc.), so this function will return the original [RID].
 			</description>
 		</method>
-		<method name="setup_local_to_scene" deprecated="This method should only be called internally. Override [method _setup_local_to_scene] instead.">
+		<method name="setup_local_to_scene" deprecated="This method should only be called internally.">
```

Comment:
Is this intentional? Is this incorrect advice?

Reply:
"Instead" doesn't make sense because it is not direct equivalence. Overriding the original `setup_local_to_scene` does nothing because it is not the virtual method.